provisioning/provisioning-service-client/pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>com.microsoft.azure.sdk.iot.provisioning</groupId>
```

Comment:
Cant comment for the deletion : Files ServiceUpdateEnrollment*.. why was that sample and readme etc deleted?

Reply:
The update operation is now included in the ServiceEnrollmentSample directly. The Readme will have to be merged though

Reply:
Compared both the readme. The ServiceEnrollmentSample readme has all of the information